.github/workflows/build-test-e2e.yml

```diff
@@ -99,10 +122,34 @@ jobs:
         run: yarn install --frozen-lockfile
       - name: Install Playwright Browsers
         run: yarn playwright install --with-deps
-      - name: Confirm Docker container is running
+      - name: Check for common errors
         run: |
```

Comment:
At some point maybe we just put this script into a file and run it. I think it's fine in this action for now

Reply:
Good call!